Documentation/boostrap-new-os.md

```diff
@@ -72,3 +72,30 @@ TODO: describe how to do debug build and replace System.Private.CoreLib.dll
 #### Using bootstrap CLI on platforms with different OS or architecture
 If the bootstrap CLI was created for OS other than the one the seed cli supports (e.g. FreeBSD) or the target architecture of the target OS differs from the architecture of the bootstrap cli (e.g. ARM64), the managed assemblies in the bootstrap CLI cannot be loaded, since they contain native code for the target OS and architecture. Fortunately, they also contain the original IL code, so they can be re-crossgened for the target OS and architecture. Or it is possible to instruct the CoreCLR runtime to ignore the native code and use the IL by setting environment variables `COMPlus_ZapDisable=1` and `COMPlus_ReadyToRun=0`. That can be useful during the bringup. But ultimately, re-crossgening should be made to improve startup performance.
 TODO: describe how to re-crossgen the managed assemblies
```

Comment:
Since you are touching this file, could you please remove this TODO? The re-crossgen was already added by @sdmaclea in #332 and it happens automatically.

Reply:
Will do, thanks.